src/components/Toast/Toast.driver.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './Toast.st.css';
+import { TOAST_SKIN } from './Toast';
+
+export interface ToastDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
+  isSuccess(): Promise<boolean>;
+  isError(): Promise<boolean>;
+  isStatus(): Promise<boolean>;
+  getCloseButton(): Promise<UniDriver>;
+  getMessage(): Promise<UniDriver>;
+  isCloseButtonExists(): Promise<boolean>;
+}
+
+const hasSkin = async (base: UniDriver, skin: TOAST_SKIN): Promise<boolean> => {
+  const skinValue = await base.attr('data-skin');
+  return skinValue === skin;
+};
+
+export const toastDriverFactory = (base: UniDriver): ToastDriver => {
+  const stylableUtil = new StylableUnidriverUtil(style);
+
+  const getCloseButton = async () => base.$('[data-hook="closeButton"]');
+  const getMessage = async () => base.$('[data-hook="message"]');
+
+  return {
+    ...baseUniDriverFactory(base),
+    isMobile: async () => stylableUtil.hasStyleState(base, 'mobile'),
```

Comment:
You can add `data-mobile` like `data-skin` and then you can remove `StylableUnidriverUtil` completely